lib/MusicBrainz/Server.pm

```diff
@@ -362,9 +362,11 @@ around dispatch => sub {
     my $unset_beta = (defined $c->req->query_params->{unset_beta} &&
                       $c->req->query_params->{unset_beta} eq '1' &&
                       !DBDefs->IS_BETA);
-    my $beta_redirect = (defined $c->req->cookies->{beta} &&
-                      $c->req->cookies->{beta}->value eq 'on' &&
-                      !DBDefs->IS_BETA);
+    my $beta_cookie = $c->req->cookies->{beta};
+    my $beta_redirect = (defined $beta_cookie &&
+                         defined $beta_cookie->value &&
```

Comment:
Same as above but for commit <https://github.com/metabrainz/musicbrainz-server/pull/2378/commits/c8486e297b31b737629735f441736f981a09a3da>: What is the legitimate case for `value` being undefined?

Reply:
Same as above :)